features/events/traps/src/test/java/org/opennms/netmgt/trapd/HibernateTrapdIpMgrIT.java

```diff
@@ -1,111 +0,0 @@
-/*******************************************************************************
```

Comment:
This test class should not be removed. It should be changed to work with the TrapdIpManagerDaoImpl.